spec/e2e/active_model_spec.rb

```diff
@@ -436,6 +436,27 @@ def true_results?(node, verb)
       expect(person.name).to eq 'Wilson'
     end
 
+    it 'can increment an attribute' do
+      person = Person.create(name: 'andreas', age: 21)
+      person.increment(:age)
+      expect(person.age).to eq(22)
```

Comment:
I think that this could be:

    expect { person.increment(:age) }.to change { person.age }.from(22).to(21)

Though if you're not using `age_was` then I don't think the dirty tracking stuff that you implemented will be tested.  It might be better to test that separately anyway.